examples/vic-gov-au/.env

```diff
@@ -0,0 +1,30 @@
+# Tide API
```

Comment:
Hey man I dont think you meant to commit this file, the .env file being removed from .gitignore was just for the reference site